tests/unit/packaging/test_models.py

```diff
@@ -780,12 +781,12 @@ def test_is_url_verified(self, db_session, url, publisher_url, expected):
         DBFileEventFactory.create(
             source=release_file,
             tag="fake:event",
-            additional={"publisher_url": publisher_url},
+            additional={"base_publisher_url": base_publisher_url},
         )
         DBFileEventFactory.create(
             source=release_file,
             tag="fake:event",
-            additional={"publisher_url": publisher_url},
+            additional={"base_publisher_url": base_publisher_url},
```

Comment:
Why do we need this second event?

Reply:
To catch the bug that this commit fixed: https://github.com/pypi/warehouse/pull/15869/commits/7f93c4226600368f8e38e8e303e23425943e1872. It was failing because it doesn't work when there are multiple events found.

Reply:
I see!! Awesome! Thanks!